### ISSUE TITLE
Create mocks for variants of appointment card in appointment list

### ISSUE THREAD

**Issue by markgreenburg** [department-of-veterans-affairs/va.gov-team#2248]
## Context
Many of these mocks probably already exist somewhere, but we need to confirm that they're up to date and we need to put them in one place so that developers can ensure they've accounted for all the variants.

## Story
As a developer, I want to know what each appointment / request variant's card should look like in the appointment list so that I can build the right thing to show to veterans.

## Variants
- Direct Scheduled appointments & confirmed VA appointment requests
    - Video
    - Office
- Confirmed Community Care appointment requests
- VA appointment requests
- Community Care appointment requests

## Available confirmed appointment data:
Facility appointment data:
- Facility id (just id, no name or anything)
- Clinic name
- Appointment time/length
- Booking note

Video appointment data:
- Facility name, id
- Video link
- Appointment time
- Provider name/contact info
- Various codes and statuses

CC appointment data:
- Provider name, phone, and address
- Veteran instructions
- Appointment time

Notably, we don't appear to have type of care for any of these cases.

## Available VA request data
- Care type
- Preferred appointment date(s) & am / pm
- Preferred facility
- Purpose of visit (e.g., new issue, routine follow-up)
- Visit type (e.g., office visit)
- Contact number & time of day preferences

## Available CC request data
- Not sure, presumably everything is the same across the two + this would perhaps return CC preferences, but we need to check with a dev first.

## AC
- [ ] Mocks for all variants exist and are attached to this ticket.

**Comment by rtwell** [comment]
here's a mock and some annotations (repeated below). I tossed the mobile version on the right hand side

1. Sort order of the list:
2. Confirmed appointments by soonest first. If a user cancels a confirmed appt, it stays in this order, but changes status. Then we list pending/canceled by…alpha? Type of care? Open to suggestions here @goldenmeanie @lalexanderson-dsva 
3. **VA facility appointments** list _both_ the Clinic as well as Facility name and address. Additionally, we list the patient friendly name for the clinic. If no patient friendly name is provided, we list the internal clinic name. 
4. We will not list “VA Facility Appointment” as an **appointment type**, however, “Community Care” and “VA Video Connect” appointment types are listed.
5. Canceled appointments no longer include a cancel button or add to calendar link
6. On all appointments, we list reason for appt and additional details (which are required, and present in both the Request and Direct schedule flows)
7. Pending appointments have an interaction to show/hide details (see comps)
8. There are 2 kinds of canceled appointments: **Requests** that have been canceled and **confirmed appointments** that have been canceled.

![image](https://user-images.githubusercontent.com/12564977/67396200-28713f80-f575-11e9-8728-99f8fef81e3b.png)
![image](https://user-images.githubusercontent.com/12564977/67396229-3030e400-f575-11e9-90e0-d61f5377c639.png)
